src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -343,6 +365,8 @@
   (mapv (partial str "DNS:")
         (utils/get-subject-dns-alt-names cert-or-csr)))
 
+(defn seq-contains? [coll target] (some #(= target %) coll))
```

Comment:
I will have to check again but think it would not work for me then which led me to this.

Reply:
In a sequence, `contains?` will check whether the index is present rather than the value, which is almost never useful. The function here is correct.
People often write
```clojure
(some #{target} coll)
```
which takes advantage of the fact that a set of one element can be used as a function that will return true when the value is matched.

Reply:
I had an alternative here but deleted it because I think it doesn't work well when the value can be something falsy.

Reply:
Cool, thanks for clearing that up!